src/theme/components/breadcrumb.ts

```diff
@@ -115,7 +115,7 @@ const Breadcrumb = defineStyleConfig({
   baseStyle: defineStyle({
     bg: "ui.black",
     color: "ui.white",
-    fontSize: "desktop.breadcrumbs.default",
+    fontSize: "desktop.body2",
```

Comment:
Is this right? @bigfishdesign13

Reply:
Specifically the default value.

Reply:
I just looked at the `typography.ts` file and I see there is a group of tokens flagged as deprecated, under which there is a `breadcrumbs` token. I think that is wrong. Let's talk about it.

Reply:
Sounds good. I remember we briefly discussed it and added it to the ticket, but I don't remember why.

Reply:
We can discuss tomorrow during standup.